keras/layers/recurrent.py

```diff
@@ -54,36 +54,56 @@ def __init__(self, cells, **kwargs):
                                  '`state_size` attribute. '
                                  'received cells:', cells)
         self.cells = cells
+        # reverse_state_order determines whether the state size will be in a
+        # reverse order of the cells' state. User might want to set this to True
+        # to keep the existing behavior. This is only useful when use
+        # RNN(return_state=True) since the state will be returned as the same
```

Comment:
Nit: use backquotes around code keywords in comments.

Reply:
Done.